ts/store/reducers/profile.ts

```diff
@@ -148,10 +149,18 @@ export const isProfileEmailValidatedSelector = createSelector(
     )
 );
 
-export const profileRemindersPreferenceSelector = createSelector(
+// return preferences
+export const profilePreferencesSelector = createSelector(
   profileSelector,
-  (profile: ProfileState): pot.Pot<boolean, Error> =>
-    pot.map(profile, p => p.reminder_status === ReminderStatusEnum.ENABLED)
+  (
+    profile: ProfileState
+  ): pot.Pot<{ reminder: boolean; preview: boolean }, Error> =>
+    pot.map(profile, p => ({
+      reminder: p.reminder_status === ReminderStatusEnum.ENABLED,
+      preview:
+        p.push_notifications_content_type ===
+        PushNotificationsContentTypeEnum.FULL
+    }))
```

Comment:
I'm not completely sure we should handle this logic in a selector, by definitions a selector should be the access point of our state, so I imagine that the selector should only return the information stored in redux state (eg: value of `reminder_status` and `push_notifications_content_type`, and then a utility function invoked by the representational component should handle the UI logic (`true`/`false` values in this case) what do you think?